devlog/_plan/260903_gemini_38_rollout/006_maintainer_review_fold.md

```
@@ -0,0 +1,56 @@
+```
+gemini-3.6-flash  --rule 0-->  gemini-3.7-flash-tiered   (a rejecting generation)
+```
+
+Retired ids deliberately keep their OWN identity in `ANTIGRAVITY_USAGE_BASE_BY_ID` — that is
+the rule protecting historical spend from being relabelled — so they can never canonicalize
+into the generation they actually call. The two mechanisms were each correct and combined into
+a hole.
+
+Probe, 2026-09-03, live CCA:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)